tests/test_monitor.py

```diff
@@ -367,9 +367,16 @@ def test_list_padding_invalid_monitor(hlwm):
 
 @pytest.mark.parametrize("mon_num,focus_idx", [
     (num, focus) for num in [1, 2, 3, 4, 5] for focus in [0, num - 1]])
-@pytest.mark.parametrize("delta", ['-1', '+1'])
-@pytest.mark.parametrize("command", ['cycle_monitor', 'focus_monitor'])
-def test_cycle_monitor(hlwm, mon_num, focus_idx, delta, command):
+@pytest.mark.parametrize("command, delta", [
+    ('cycle_monitor', '-1'),
+    ('cycle_monitor', '+1'),
+    ('cycle_monitor', '1'),
```

Comment:
This is another new testcase, albeit unrelated to the issue I think it makes sense to test this too.